manim/animation/indication.py

```diff
@@ -65,29 +71,27 @@ def create_target(self):
         return target
 
 
+@attr.s(auto_attribs=True, eq=False)
 class Flash(AnimationGroup):
-    CONFIG = {
-        "line_length": 0.2,
-        "num_lines": 12,
-        "flash_radius": 0.3,
-        "line_stroke_width": 3,
-        "run_time": 1,
-    }
-
-    def __init__(self, point, color=YELLOW, **kwargs):
-        self.point = point
-        self.color = color
-        digest_config(self, kwargs)
+    line_length: float = 0.2
+    num_lines: int = 12
+    flash_radius: float = 0.3
+    line_stroke_width: float = 3
```

Comment:
this is clearly not a float
```suggestion
    line_stroke_width: tp.Union[int, float] = 3
```